RedHat-UX/red-hat-design-system
# docs(card): update Style subpage

## What I did

Made the following updates to card's [Theme section](https://deploy-preview-1664--red-hat-design-system.netlify.app/elements/card/style/#theme):
- Added text for the Theme section
- Added headings for light and dark theme sections
- Updated images for light and dark themes
- Added copy for Color section
- Changed Color examples from images to `<rh-card>` elements


## Testing Instructions

Check the [card's Theme section](https://deploy-preview-1664--red-hat-design-system.netlify.app/elements/card/style/#theme) in the DP.

## Notes to Reviewers
It looks like the site's style.css file is adding a margin above all `<p>` tags. This is a known issue with `<rh-card>` and should be tackled in a separate issue/PR.

<img width="1435" alt="Screenshot 2024-06-26 at 2 57 39 PM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/0ddff943-5e33-45da-b4df-5b3a242b6604">
